client/stylesheets/_toCat.scss

```diff
@@ -3116,18 +3126,47 @@ On click the images will be displayed at normal size to complete the effect
 
 .listingItemDetailItems {
     margin: 0;
-    margin-top: 5px;
-    padding-left: 0.5em;
-    font-size: 1em;
+    padding: 1rem;
+    font-size: 1rem;
 }
 
-.listingItemDetailItems li {
-    margin: 8px 0;
-    font-size: 1.1rem; 
+.listingItemDetailItems > li {
+    margin: 0;
+    display: inline-block;
 }
 
-.listingItemDetailItems li a:hover {
-    color: #5d5d5d;
+.listingItemDetailItems li ul li a {
+    margin-left: 1rem;
+    color: #404040 !important;
+}
+
+.listingItemDetailItems li ul li a:hover {
+    color: #dd4b39 !important;
+}
+
+.listingItemDetailRight {
+    width: 45%;
+}
+
+.listingItemDetailLeft {
+    width: 45%;
+}
+
+.listingItemDetailLeft ul li:nth-child(2), .listingItemDetailRight ul li:nth-child(2) {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `listingItemDetailLeft` should be written in lowercase with hyphens<br>Selector `listingItemDetailRight` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line